client/src/app/modules/module-instance/components/fields/date-field/date-field.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-form-field>
   <input matInput [matDatepicker]="picker" [placeholder]="cData.label" [formControl]="cData.control">
   <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
-  <mat-datepicker #picker startView="year" [startAt]="startDate"></mat-datepicker>
+  <mat-datepicker [touchUi]="cData['touchUi']" #picker [startView]="cData['startView']" [startAt]="cData['startAt'] ? cData['startAt'] : dateNow | date "></mat-datepicker>
```

Comment:
Do we need to apply the `date` pipe here? I also think it would be better to implement a `startAt` property that is defined in the `onInit` pipe